vendorsetup.sh

```diff
@@ -30,14 +30,15 @@ function lunch
     if [[ $(find device/intel -path "*mixins*" -prune -o -name ${TARGET_PRODUCT}.mk -print) ]]; then
         product_dir=$(dirname $(find device/intel -path "*mixins*" -prune -o -name ${TARGET_PRODUCT}.mk -print))
 	echo "Executing mixin update..."
-        mixinup -s $product_dir/mixins.spec
+        #mixinup -s $product_dir/mixins.spec
+        mixinup
     fi
 }
 
 # Get the exact value of a build variable.
 function get_build_var()
 {
-    if [ "$1" = "COMMON_LUNCH_CHOICES" ]
```

Comment:
I think its a typo ?

Reply:
Its a temporary change, ill remove this function from script soon